web/src/pages/profile.tsx

```diff
@@ -433,7 +433,7 @@ const ProfilePage = (props: Props): ReactElement => {
               fieldName="homeBasedBusiness"
               isVisible={displayHomedBaseBusinessQuestion()}
               displayAltDescription={displayAltHomeBasedBusinessDescription}
-              compact
+              hideLine
```

Comment:
There were multiple competing style props, I just consolidated these. See  `ProfileField.tsx` for full list.